packages/hardhat/test/integration-tests/periphery/controller-helper.ts

```diff
@@ -105,30 +105,31 @@ describe("Controller helper integration test", function () {
   describe("Mint short with flash deposit", async () => {
     it("flash mint", async () => {      
       const vaultId = await shortSqueeth.nextId();
-      // await controller.connect(depositor).updateOperator(vaultId, controllerHelper.address)
 
       const normFactor = await controller.normalizationFactor()
-      const mintWSqueethAmount = ethers.utils.parseUnits('2')
+      const mintWSqueethAmount = ethers.utils.parseUnits('10')
       const mintRSqueethAmount = mintWSqueethAmount.mul(normFactor).div(one)
       const ethPrice = await oracle.getTwap(ethDaiPool.address, weth.address, dai.address, 420, true)
       const scaledEthPrice = ethPrice.div(10000)
       const debtInEth = mintRSqueethAmount.mul(scaledEthPrice).div(one)
       const collateralAmount = debtInEth.mul(3).div(2).add(ethers.utils.parseUnits('0.01'))
-      const value = collateralAmount.div(2).add(collateralAmount.div(3))
+      const squeethEthPrice = await oracle.getTwap(wSqueethPool.address, wSqueeth.address, weth.address, 420, true)
+      const ethAmountOut = mintWSqueethAmount.mul(squeethEthPrice).div(one)
```

Comment:
i still think its worth checking the exact expected out here (ie including price impact on uniswap), but lmk what you think

that way we can compare and ensure that the user is refunded value - ethAmountOut, which means that 0 of the slippage was used (which is to be expected when testing)